configs/devnet/docker-setup-config.yaml

```diff
@@ -2,19 +2,19 @@ defaultStake: 10000
 defaultFee: 2000
 borChainId: 15005
 heimdallChainId: heimdall-4052
-borBranch: develop
-heimdallBranch: develop
+borBranch: v0.3.0-beta5
+heimdallBranch: v0.3.0-beta2
 contractsBranch: jc/v0.3.1-backport
 sprintSize: 64
 blockNumber: '0'
 blockTime: '2'
-numOfValidators: 1
+numOfValidators: 2
 numOfNonValidators: 1
 ethURL: http://ganache:9545
 ethHostUser: ubuntu
 devnetType: docker
 borDockerBuildContext: https://github.com/maticnetwork/bor.git#develop
 heimdallDockerBuildContext: https://github.com/maticnetwork/heimdall.git#develop
-devnetBorUsers: ubuntu,ubuntu
+devnetBorUsers: ubuntu,ubuntu,ubuntu
 devnetBorHosts:
-  - 35.92.248.232
+  - 35.86.96.17
```

Comment:
We do not want to push `docker-setup-config.yaml` and `remote-setup-config.yaml`

Reply:
Reverted : [cd99f61](https://github.com/maticnetwork/matic-cli/pull/82/commits/cd99f615f46356bf34d794d0e2e8dc50aafbdb48)